.isort.cfg

```diff
@@ -0,0 +1,6 @@
+[settings]
+py_version=37
+line_length=100
+use_parentheses=True
+multi_line_output=5
+add_imports=from __future__ import annotations
```

Comment:
> Finally, I've configured isort to automatically add `from __future__ import annotations` to all python files. This has found a number of places it's missing, and allows us to catch this in a linter instead of having to manually inspect for that.

This seems wrong. I catch this and manually inspect it with my linter already (not isort). There are a couple types of files that don't have this import:

- `__init__.py` solely used to import symbols
- `_typing.py` and similar files that exist solely to be imported by the type checker
- exactly three files that don't contain any type annotations

In general, missing this future import is a fatal runtime error because I also scan for typing imports that should only be imported inside of `T.TYPE_CHECKING` and have a runtime dependency on using the future import.